examples/simple/igraph_community_leiden.c

```diff
@@ -25,7 +25,7 @@
 
 int main() {
     igraph_t graph;
-    igraph_vector_t membership, degree;
+    igraph_vector_int_t membership, degree;
```

Comment:
I see that the `degree` is now also changed to an `igraph_vector_int_t`, but the initialization is still done using `igraph_vector_init` and `igraph_degree` still uses a `igraph_vector_t` instead of an `igraph_vector_int_t`. Presumably best to leave this as an `igraph_vector_t` and change it when we change `igraph_degree` to us an `igraph_vectort_int_t` (if we so decide)?

Reply:
Yes, I was just a bit sloppy there. I will change it back and maybe handle degrees later.